src/mono/sample/Android/AndroidSampleApp.csproj

```diff
@@ -24,6 +24,21 @@
       <ApkDir>$(PublishDir)apk\</ApkDir>
     </PropertyGroup>
 
+    <ItemGroup>
+      <_CommonLinkerArgs Include="libz.so" />
+      <_CommonLinkerArgs Include="log" />
+      <_CommonLinkerArgs Include="-u GlobalizationNative_LoadICU" />
```

Comment:
this duplication is unfortunate, all the more reasons to figure out why we need the `-u`